src/token/erc20/erc20.cairo

```diff
@@ -12,11 +12,11 @@ mod ERC20 {
 
     #[storage]
     struct Storage {
-        _name: felt252,
-        _symbol: felt252,
-        _total_supply: u256,
-        _balances: LegacyMap<ContractAddress, u256>,
-        _allowances: LegacyMap<(ContractAddress, ContractAddress), u256>,
+        ERC20_name: felt252,
+        ERC20_symbol: felt252,
+        ERC20_total_supply: u256,
+        ERC20_balances: LegacyMap<ContractAddress, u256>,
+        ERC20_allowances: LegacyMap<(ContractAddress, ContractAddress), u256>,
```

Comment:
are we positive `LegacyMap` is structured exactly the same as
```
func ERC20_allowances(owner: felt, spender: felt) -> (remaining: Uint256)
```
?

Reply:
Good question, I think we should confirm with Starkware

Reply:
Confirmed!